client/src/components/reviews/SortBy.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import axios from 'axios';
+import { TOKEN } from '../../../../config.js';
+
+function SortBy({ setCurrentReviews }) {
+
+  const handleOnChange = e => {
+    const selected = e.target.value;
+    const url = `https://app-hrsei-api.herokuapp.com/api/fec2/hr-nyc/reviews/?sort=${selected}&product_id=39333`
```

Comment:
Consider pulling the API request out of this file and into the `request.js` file